service/src/main/java/com/codeforcommunity/requester/S3Requester.java

```diff
@@ -162,7 +167,7 @@ private static String validateBase64ImageAndUploadToS3(
             externs.getBucketPublic(), directoryName + "/" + fullFileName, tempFile);
 
     // Set the image to be publicly available
-    awsRequest.setCannedAcl(CannedAccessControlList.PublicRead);
+    // awsRequest.setCannedAcl(CannedAccessControlList.PublicRead);
```

Comment:
Commenting this out for now until we figure out S3 permissions (probably what I'll do next)